### PR TITLE
Added state for Multimedia Audio or Visual

### PR DESCRIPTION
**Pull request by NidhiHemanth**
Added state for Multimedia Audio or Visual.

### CODE REVIEW

**Comment by NidhiHemanth** [comment]
@sn99

File: src/operating_system/multimedia_audio_visual.rs [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    /// 
+    /// `Video` ("Video")
+    pub Group: Option<String>,
+    /// If `True`, the file is hidden.
+    pub Hidden: Option<bool>,
+    /// Object was installed. This property does not require a value to indicate that the object is installed.
+    pub InstallDate: Option<WMIDateTime>,
+    /// Number of "file opens" that are currently active against the file.
+    /// 
+    /// For more information about using `uint64` values in scripts, see Scripting in WMI.
```
Comment (sn99) [inline src/operating_system/multimedia_audio_visual.rs:131]: Remove this line

**Review by sn99 — CHANGES_REQUESTED** [review]

File: src/operating_system/multimedia_audio_visual.rs (hunk 1 above)
Comment (sn99) [reply src/operating_system/multimedia_audio_visual.rs:131]: @NidhiHemanth

**Comment by NidhiHemanth** [comment]
@sn99 done

**Comment by NidhiHemanth** [comment]
@sn99 should i remove all such lines?

**Comment by sn99** [comment]
> @sn99 should i remove all such lines?

@NidhiHemanth I don't think there were any in other prs but yeah in future anything irrelevant can be removed.

**Pull request merged**